services/arrow-flight/src/main/java/com/dremio/service/flight/impl/FlightPreparedStatement.java

```diff
@@ -65,6 +67,22 @@ public FlightInfo getFlightInfo(Location location) {
     return new FlightInfo(schema, flightDescriptor, ImmutableList.of(flightEndpoint), -1, -1);
   }
 
+  public ActionCreatePreparedStatementResult createAction() {
+    final UserProtos.CreatePreparedStatementArrowResp createPreparedStatementResp = responseHandler.get();
+    final Schema schema = buildSchema(createPreparedStatementResp.getPreparedStatement().getArrowSchema());
+
+    return ActionCreatePreparedStatementResult.newBuilder()
+        .setDatasetSchema(ByteString.copyFrom(schema.toByteArray()))
+        .setParameterSchema(ByteString.EMPTY)
```

Comment:
Is this true?

Reply:
Yes, FlightPreparedStatement doesn't support parameters